build/keyboard-layouts.js

```diff
@@ -89,7 +89,7 @@ function genDefaultLayouts(config, webappsMapping, allLayouts) {
 
 function execute(options) {
   // Get web apps mapping file
-  let stageFolder = utils.getEnv('STAGE_DIR');
+  let stageFolder = options.STAGE_DIR;
```

Comment:
We are unable to get / inherit environment variables from parent process like as in node.js since it doesn't implemented on XPCOM yet. Thus, just options.STAGE_DIR here.

see also https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/system_child_process